docs/api.md

```diff
@@ -0,0 +1,40 @@
+Short API doc.
+
+# Authentication
```

Comment:
Since we decided to develop a telegram bot, we need a way to authorize a telegram user. Does it make sense to create methods to grant and revoke access for a telegram user by token?